esp-hal-common/src/usb_serial_jtag.rs

```diff
@@ -192,6 +200,15 @@ pub trait Instance {
 
         (wr_addr - rd_addr).into()
     }
+
+    fn txfifo_empty(&self) -> bool {
+        !self
+            .register_block()
+            .jfifo_st
+            .read()
+            .out_fifo_empty()
+            .bit_is_set()
```

Comment:
```suggestion
        self
            .register_block()
            .jfifo_st
            .read()
            .out_fifo_empty()
            .bit_is_clear()
```

Not a huge deal, but just a bit more clear IMO.

Reply:
Fixed